packages/jazz-tools/src/react-core/subscription-provider.tsx

```diff
@@ -42,24 +44,25 @@ export function createCoValueSubscriptionContext<
         resolve: resolve,
       });
 
-      const loadState = useSubscriptionSelector(subscription, {
-        select: (value) => (!value ? value : true),
-      });
+      const loadState = useSubscriptionSelector(subscription);
```

Comment:
```suggestion
      const loadState = useSubscriptionSelector(subscription, {
        select: (value) => value.$jazz.loadingState,
      });
```

Otherwise, the entire tree will re-render on changes already loaded values